### ISSUE TITLE
Cannot install azure-functions-core-tools on macOS 13 with Homebrew

### ISSUE THREAD

**Issue by agoncal** [Azure/azure-functions-core-tools#3189]
Hi

I'm running on macOS 13.0 (Ventura) and when I try to install azure-functions-core-tools I have the following error:

```
$ brew install azure-functions-core-tools

==> Downloading https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.osx-x64.2.7.3188.zip
/Users/agoncal/Library/Caches/Homebrew/downloads/c52e7540126f94a2d340e0cc04501cb551a53f13bebcd2b7633348e1316206e2--Azure.Functions.Cli.osx-x64.2.7.3188.zip
==> Installing azure-functions-core-tools from azure/functions
Error: Your Command Line Tools (CLT) does not support macOS 13.
```

It looks like the Azure Functions tools are not ready for Ventura, correct?

Here are more details about my environment:

```
$ uname -a
Darwin Kernel Version 22.1.0

$ sw_vers
ProductName:		macOS
ProductVersion:		13.0
BuildVersion:		22A380

$ brew --version
Homebrew 3.6.7
Homebrew/homebrew-core (git revision ff1d6c2f855; last commit 2022-10-26)
Homebrew/homebrew-cask (git revision a2aa94513b; last commit 2022-10-26)
```

**Comment by csalgo70** [comment]
I am having the same issue on Ventura.

**Comment by petertuton** [comment]
Workaround here: https://github.com/orgs/Homebrew/discussions/3822#discussioncomment-3965318 

TLDR; install Xcode 14.1 RC2, or just Command Line Tools for Xcode 14.2 if you don't need full-blown Xcode.

**Comment by agoncal** [comment]
Works fine with 14.2
